### ISSUE TITLE
Tracking issue for [slatepack]

### ISSUE THREAD

**Issue by lehnberg** [mimblewimble/grin-wallet#406]
This is a tracking issue for RFC000X, tentative RFC Proposal https://github.com/mimblewimble/grin-rfcs/pull/55, "Slatepack", yet to be accepted.

```
This is a placeholder issue for an RFC that has yet to be accepted.
This issue will be updated if the RFC becomes accepted.
```

Once all the above have been implemented and deployed we can close this tracking issue.

**Comment by lehnberg** [comment]
Related pull request: https://github.com/mimblewimble/grin-wallet/pull/400

**Comment by quentinlesceller** [comment]
Considering that the compact slate RFC has been merged =>mimblewimble/grin-rfcs#55
And that the corresponding code have been also merged =>  https://github.com/mimblewimble/grin-wallet/pull/410 https://github.com/mimblewimble/grin-wallet/pull/411 https://github.com/mimblewimble/grin-wallet/pull/413 https://github.com/mimblewimble/grin-wallet/pull/420 https://github.com/mimblewimble/grin-wallet/pull/421 https://github.com/mimblewimble/grin-wallet/pull/423 https://github.com/mimblewimble/grin-wallet/pull/428 https://github.com/mimblewimble/grin-wallet/pull/436 https://github.com/mimblewimble/grin-wallet/pull/445 https://github.com/mimblewimble/grin-wallet/pull/446 (yeah that's a LOT of PRs 😄 )

I'm closing this issue. Awesome work @j01tz, @yeastplume and anyone who contributed, helped and tested ! 🎆
